Solutions/MimecastAudit/Data/Solution_MimecastAudit.json

```diff
@@ -0,0 +1,20 @@
+{
+    "Name": "MimecastAudit",
+    "Author": "Mimecast - dlapi@mimecast.com",
+    "Logo": "<img src=\"https://raw.githubusercontent.com/mimecast/Azure-Sentinel/mimecast_audit_connector/Solutions/MimecastAudit/Data%20Connectors/Logo/Mimecast.svg\" width=\"75px\" height=\"75px\">",

```

Comment:
The logo link should have the master branch, which is currently mimecast_audit_connector.
Please switch it to master.